### PR TITLE
Multiple fixes to README.md

### PR DESCRIPTION
**Pull request by xSavitar**
Fixes to some sections and text issues including typos in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,10 @@ Clone the **DIT** repo and build it.
   $ git clone https://github.com/openmrs/openmrs-module-dataimporttool.git
   $ cd openmrs-module-dataimporttool
   $ mvn clean install
-
-  
-  This command generates an **omod** file for the openmrs-module-data-importtool 
 ````
 
+These commands generates an **omod** file for the openmrs-module-data-importtool.
```
Comment (dkayiwa) [inline README.md]: There is a grammar error in the above sentence.

File: README.md [hunk 2]
```diff
@@ -32,8 +31,8 @@ I.) Create a dummy database, populated with data.
 
 Clone dmt-guide
 ````
-	$git clone https://github.com/esaude/dmt-guide
```
Comment (dkayiwa) [inline README.md:35]: I find it better with the indention as it was than without them.

Comment (xSavitar) [reply README.md:35]: Okay! So you would prefer `$git ...` than `$ git ...`?

Comment (dkayiwa) [reply README.md:35]: Oh, i think you were right 😊

**Comment by dkayiwa** [comment]
Thanks @ch3nkula 👍

File: README.md (hunk 2 above)
Comment (xSavitar) [reply README.md:35]: Ohh great 👍

**Comment by xSavitar** [comment]
You are welcome @dkayiwa. 👍

**Pull request merged**